models/build.js

```diff
@@ -85,6 +86,12 @@ const MODEL = {
         .description('Status message to describe status of the build')
         .example('Build failed due to infrastructure error'),
 
+    statusMessageType: Joi.string()
+        .valid(...STATUS_MESSAGE_TYPES)
```

Comment:
should we set `default`?

Reply:
This field is optional/nullable. Not all builds have `statusMessage` set. So, is the `statusMessageType`.
We don't need to assign a default value.